client/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -248,6 +248,17 @@ describe('FeedbackGeneratorService', function() {
     });
   });
 
+  describe('_getUnfamiliarLanguageFeedback', function() {
+    it('should return the string for the text feedback', function() {
+      var feedbackString = FeedbackGeneratorService
+        ._getUnfamiliarLanguageFeedback();
+      expect(feedbackString).toEqual([
+        "Seems like you're having some trouble with this language. Why ",
+        "don't you take a look at the primer?"
+      ].join(''));
+    });
+  });
+
```

Comment:
We'll have to add some tests for `resetCounters` and also to double check that the correct feedback is appended when language unfamiliarity is detected. We can pair program that part together if you want.